shared/package.json

```diff
@@ -59,6 +59,7 @@
     "pod-install-new": "cd ios; RCT_NEW_ARCH_ENABLED=1 pod install --repo-update",
     "android-clean": "cd android; ./gradlew clean",
     "android-debug": "react-native run-android --mode 'debug'",
+    "android-install-downgrade": "adb install -r -d android/app/build/outputs/apk/debug/app-debug.apk",
```

Comment:
lets us force a deploy to a local device even if the version goes down, aka this happens all the time since we use the git commit length